### ISSUE TITLE
Syntax highlighting worse with some color themes after #1337?

### ISSUE THREAD

**Issue by J0F3** [PowerShell/PowerShellEditorServices#1368]
Hi,

In the preview version of the VS Code PowerShell extension the syntax highlighting seems to be worse for many color themes than it is currently in the stable release. I am not quite sure what it is but I assume it was the change in #1337.

Version v2020.6.0:
One Dark Pro:
![image](https://user-images.githubusercontent.com/12759579/95837296-bcb55300-0d40-11eb-99ac-80aebf273a7d.png)
Nord:
![image](https://user-images.githubusercontent.com/12759579/95837251-b0c99100-0d40-11eb-9e3d-f486f4a15604.png)

Version v2020.9.0 (Preview):
One Dark Pro:
![image](https://user-images.githubusercontent.com/12759579/95836036-4106d680-0d3f-11eb-891d-d6d167280d84.png)
Nord:
![image](https://user-images.githubusercontent.com/12759579/95835939-27fe2580-0d3f-11eb-82ff-a6c351819a7e.png)

Thanks for looking into it!
(I am not sure if I am at the correct place here but I saw the change mentioned above here so I posted it here for now.)

**Comment by rjmholt** [comment]
See https://github.com/PowerShell/vscode-powershell/issues/2852#issuecomment-669259784

**Comment by J0F3** [comment]
While I not quite understand the cause and what is discussed in #2852 (which is not your problem though 😉) I see that apparently there is already work ongoing regarding this.
So you may close this one if you like.

**Comment by rjmholt** [comment]
> While I not quite understand the cause and what is discussed in #2852

Essentially the issue is that "accurate token categorisation" and "better color highlighting" aren't actually 100% aligned. Even though we categorise tokens better, because of the way colourschemes work we face issues with the colours ending up looking worse.

I'll close this issue as a duplicate of the other, since we already have some discussions about semantic highlighting.

**Comment by rjmholt** [comment]
Duplicate of https://github.com/PowerShell/vscode-powershell/issues/2852
